CHANGELOG.md

```diff
@@ -30,6 +30,7 @@
 ### 🧰 Bug fixes 🧰
 
 * Removed additional dependencies specified in featurebyte client
+* Fixed bug in `feature.definition` so that it is consistent with the underlying query graph
```

Comment:
Since 0.2.2 is already out, I think this should be moved up to the unreleased version section.

Reply:
Good catch! Will fix this.